File: CPlusPlus/5597.cpp
#include <bits/stdc++.h>

#define FASTIO                             \
    std::ios_base::sync_with_stdio(false); \
    std::cin.tie(0);                       \
    std::cout.tie(0)
#define ENDL '\n'

/***************************************************************************************
 *   * 링크 : https://www.acmicpc.net/problem/5597
 ***************************************************************************************/

using namespace std;

int n;
bool arr[32];

int main() {
    FASTIO;
    for (int i = 0; i < 28; i++) {
        cin >> n;
        arr[n] = true;
    }

    for (int i = 1; i <= 30; i++) {
        if (!arr[i]) {
            cout << i << ENDL;
        }
    }

    return 0;
}

File: CSharp/5597.cs
using System;
using System.Text;

// 링크 : https://www.acmicpc.net/problem/5597

namespace BOJ
{
    internal class Program
    {
        static void Main()
        {
            bool[] arr = new bool[31];
            StringBuilder sb = new StringBuilder();

            for (int i = 0; i < 28; i++)
            {
                int student = int.Parse(Console.ReadLine());

                arr[student] = true;
            }

            for (int i = 1; i <= 30; i++)
            {
                if (!arr[i])
                {
                    sb.AppendLine(i.ToString());
                }
            }

            Console.Write(sb);
        }
    }
}